wdl/gwas/regenie_step1.wdl

```diff
@@ -12,13 +12,74 @@ task step1 {
     String options
 
     String docker
-
+    Int cov_filter_th = 10
```

Comment:
Should we have this as input in the json just to have it more transparent

Reply:
Yeah probably